db/schema.rb

```diff
@@ -88,6 +88,19 @@
   add_index "bundle_products", ["bundle_product_id"], name: "fk_bundle_prod_prod", using: :btree
   add_index "bundle_products", ["product_id"], name: "fk_bundle_prod_bundle", using: :btree
 
+  create_table "card_readers", force: :cascade do |t|
```

Comment:
I'm wondering if we should prefix these with `secure_room_`. Part of me feels we should do the model classes could be namespaced (`SecureRooms::CardReader`). We do the namespace part for Projects (the model is `Projects::Project`), which I confess is a little awkward, much like `SecureRooms::Room` would have been. What do you think?